exporter/signalfxexporter/README.md

```diff
@@ -81,6 +81,8 @@ The following configuration options can also be configured:
 that are allowed to be used as a dimension key in addition to alphanumeric 
 characters. Each nonalphanumeric dimension key character that isn't in this string 
 will be replaced with a `_`.
+- `max_connections` (default = 100): MaxConnections is used to set a limit to the maximum
```

Comment:
if this parameters is used to set the max number of idle connections I think that it might make more sense to name it max_idle_connections

Reply:
I have taken a reference of [sapm](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/sapmexporter/config.go#L49) exporter, Which has the same parameter for the same purpose.

Reply:
okay, makes sense, thanks

Reply:
I can see it either way, but if we want to enforce this overall it could additionally be used to set the `MaxConnsPerHost` transport option.